api/controllers/TagController.js

```diff
@@ -1,19 +1,91 @@
+var fetchAndPresentFollowed = (communityId, userId) =>
+  FollowedTag.where({community_id: communityId, user_id: userId})
+  .fetchAll({withRelated: 'tag'})
+  .then(followedTags =>
+    followedTags.map(followedTag => ({
+      name: followedTag.relations.tag.get('name')
+    })))
+
+var fetchAndPresentCreated = (communityId, userId) =>
+  CommunityTag.where({community_id: communityId, owner_id: userId})
+  .fetchAll({withRelated: 'tag'})
+  .then(createdTags =>
+    createdTags.map(createdTag => ({
+      name: createdTag.relations.tag.get('name')
+    })))
+
 module.exports = {
 
   findOne: function (req, res) {
     return Tag.find(req.param('tagName'))
     .then(tag => {
       if (!tag) return res.notFound()
       return CommunityTag.where({community_id: res.locals.community.id, tag_id: tag.id})
-      .fetch({withRelated: 'owner'})
+      .fetch({withRelated: ['owner', 'community.followedTags']})
       .then(communityTag => {
         var result = communityTag.pick('id', 'description', 'community_id')
         result.name = tag.get('name')
         result.owner = communityTag.relations.owner.pick('id', 'name', 'avatar_url')
+        result.followed = !!communityTag.relations.community.relations.followedTags.find(ft => {
```

Comment:
@levity - this find seems like something I should be doing in sql, but I'm not sure how to add those extra constraints. Is there a way to parameterize the bookshelf community.followedTags method to take user_id and tag_id?